Web/starter_project_barbies/styles/globals.css

```diff
@@ -5,7 +5,7 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
+/* :root {
```

Comment:
If it's not being used, I would suggest removing it rather than commenting out.

Reply:
done